src/GovUk.Education.ExploreEducationStatistics.Common.Tests/Extensions/GeneratorTupleExtensions.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using GovUk.Education.ExploreEducationStatistics.Common.Tests.Fixtures;
+
+namespace GovUk.Education.ExploreEducationStatistics.Common.Tests.Extensions;
+
+/// <summary>
+/// Extension methods for obtaining various sizes of Tuples from a <see cref="Generator{T}"/>.
+/// </summary>
+public static class GeneratorTupleExtensions
```

Comment:
As per my PR description comment, I was happy to house them in either place.  I moved them here because I thought you might prefer the Generator class to not have Tuple methods filling it up (although I wouldn't imagine we'd create too many of them).